CorsixTH/Lua/languages/english.lua

```diff
@@ -306,6 +306,7 @@ install = {
 
 misc.not_yet_implemented = "(not yet implemented)"
 misc.no_heliport = "Either no diseases have been discovered yet, or there is no heliport on this map. It might be that you need to build a reception desk and hire a receptionist"
+misc.cant_treat_emergency = "Your hospital cannot treat this emergency because its disease has not been discovered. Feel free to try again, cheater."
```

Comment:
I find `cheater` offensive in that sentence.
I am sure you are trying to be funny, but the reader can't see your big smile, and may mis-interpret the message.

Reply:
I was on the edge about it. Chances are it's my british humour taking over generic humour. I'll just leave it without that word :)